src/flp/types.rs

```diff
@@ -11,6 +11,9 @@ use std::fmt::{self, Debug};
 use std::marker::PhantomData;
 use subtle::Choice;
 
+#[cfg(feature = "experimental")]
```

Comment:
Do we still want to put DP stuff behond the experimental flag? I guess the APIs haven't quite stabilized, which is one argument for doing it.

Reply:
TypeWithNoise and AggregatorWithNoise are currently behind this feature, so this was the easiest change for now. I think removing the feature gate can wait for a later change.